articles/client-platforms/angularjs/00-intro.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Introduction
+description: test
+---
+
+
+
+<%= include('../../_includes/_signup') %>
+
+Auth0 provides wrappers for using the [Lock widget](https://auth0.com/lock) and the [auth0.js](https://github.com/auth0/auth0.js) library in Anguar apps, and these wrappers will be used throughout these quickstart guides.
+
+## Samples Projects
+
+If you want to follow along with these quickstart guides, download the [seed project](https://github.com/auth0-samples/auth0-angularjs-sample/tree/master/00-Starter-Seed) as a starting point. The seed project is just a basic, blank Angular application with all the Bower dependencies included and the required references added to the `index.html` file. You can also download the source code for each quickstart step from these docs.
+
+## Prerequisites
+
+Before starting, it is important to ensure your application is set up correctly by following these steps:
+
+### 1. Create an Application
+
+<%= include('../../_includes/_new_app') %>_
+
+### 2. Configure Callback URLs
+
+A callback URL is a URL in your application where Auth0 redirects to after the user has authenticated. You can set the Callback URL by going to the [Application Settings](${uiAppSettingsURL}) section of your Auth0 dashboard and make sure that **Allowed Callback URLs** contains the following value:
+
+<pre><code>http://localhost:3000/</pre></code>
```

Comment:
Can we add a comment that says the samples use `locahost:3000` and that this shouldbe placed as a callback URL? And some wording to reflect that if they are using a different host:port for their app, they need to put the appropriate one.